### PR TITLE
Feat: Adding kube-prometheus plugin

### PR DESCRIPTION
**Pull request by daniel-hutao**
Signed-off-by: Daniel Hu <tao.hu@merico.dev>

# Summary

Adding kube-prometheus plugin

## Key Points

- [x] Documentation (new or existing) is updated.

### CODE REVIEW

**Comment by daniel-hutao** [comment]
image

image

image

**Comment by daniel-hutao** [comment]
image


image

File: cmd/kubeprometheus/main.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+// Plugin is the type used by DevStream core. It's a string.
+type Plugin string
+
+// Install implements the installation of some GitHub Actions workflows.
+func (p Plugin) Install(options *map[string]interface{}) (bool, error) {
+	return kubeprometheus.Install(options)
+}
+
+// Reinstall implements the installation of some GitHub Actions workflows.
```
Comment (IronCore864) [inline cmd/kubeprometheus/main.go:20]: The comment and the function name don't match; same below.

**Review by IronCore864 — CHANGES_REQUESTED** [review]

**Review by lfbdev — APPROVED** [review]
/lgtm

**Review by IronCore864 — APPROVED** [review]

**Pull request merged**